lab_2/main.py

```diff
@@ -374,4 +388,17 @@ def tokenize_big_file(path_to_file: str) -> tuple:
     :param path_to_file: a path
     :return: a tuple with ids
     """
-    pass
+    tokens = []
+    idx = {}
+
+    file = (tokenize(row) for row in open(path_to_file, encoding='utf-8'))
```

Comment:
the point is that you read a part of a file, tokenize it, transform into a numeric form and only then read another part
there should be only one large tuple stored (with ids), not two (with tokens and ids)